Pep8 errors fail travis.

>  pep8 mycroft
mycroft/__version__.py:1:8: E225 missing whitespace around operator
mycroft/version/__init__.py:33:80: E501 line too long (83 > 79 characters)
mycroft/version/__init__.py:38:80: E501 line too long (89 > 79 characters)
mycroft/skills/core.py:142:21: E127 continuation line over-indented for visual indent
mycroft/skills/configuration/__init__.py:57:80: E501 line too long (95 > 79 characters)
mycroft/skills/wolfram_alpha/__init__.py:182:39: E128 continuation line under-indented for visual indent

--- a/mycroft/client/wifisetup/main.py
+++ b/mycroft/client/wifisetup/main.py
@@ -520,7 +520,7 @@ def main():
     try:
         wifi.run()
     except Exception as e:
-        print (e)
+        print(e)
     finally:
         sys.exit()
 

--- a/mycroft/skills/configuration/__init__.py
+++ b/mycroft/skills/configuration/__init__.py
@@ -54,7 +54,8 @@ class ConfigurationSkill(ScheduledSkill):
             self.update()
         except HTTPError as e:
             if e.response.status_code == 401:
-                self.log.warn("Impossible to update configuration because device isn't paired")
+                self.log.warn("Impossible to update configuration because "
+                              "device isn't paired")
         self.schedule()
 
     def update(self):

--- a/mycroft/skills/core.py
+++ b/mycroft/skills/core.py
@@ -138,8 +138,8 @@ def load_skills(emitter, skills_root=SKILLS_BASEDIR):
             load_skill(skill, emitter)
 
     for skill in skills:
-        if (skill['name'] not in PRIMARY_SKILLS and
-                    skill['name'] not in BLACKLISTED_SKILLS):
+        if (skill['name'] not in PRIMARY_SKILLS and skill['name'] not in
+                BLACKLISTED_SKILLS):
             load_skill(skill, emitter)
 
 

--- a/mycroft/skills/wolfram_alpha/__init__.py
+++ b/mycroft/skills/wolfram_alpha/__init__.py
@@ -179,7 +179,7 @@ class WolframAlphaSkill(MycroftSkill):
             if len(others) > 0:
                 self.speak_dialog('others.found',
                                   data={'utterance': utterance, 'alternative':
-                                      others[0]})
+                                        others[0]})
             else:
                 self.speak_dialog("not.understood", data={'phrase': phrase})
 

--- a/mycroft/version/__init__.py
+++ b/mycroft/version/__init__.py
@@ -30,10 +30,12 @@ class VersionManager(object):
 
     @staticmethod
     def get():
-        if exists(VersionManager.__location) and isfile(VersionManager.__location):
+        if (exists(VersionManager.__location) and
+                isfile(VersionManager.__location)):
             try:
                 with open(VersionManager.__location) as f:
                     return json.load(f)
             except:
-                LOG.error("Failed to load version from '%s'" % VersionManager.__location)
+                LOG.error("Failed to load version from '%s'"
+                          % VersionManager.__location)
         return {"coreVersion": None, "enclosureVersion": None}

